.vib/kubernetes-event-exporter/vib-verify.json

```diff
@@ -15,6 +15,48 @@
           "action_id": "helm-lint"
         }
       ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/kubernetes-event-exporter"
+        },
+        "runtime_parameters": "Y29uZmlnOgogIGxvZ0xldmVsOiBkZWJ1ZwogIGxvZ0Zvcm1hdDogcHJldHR5CiAgcmVjZWl2ZXJzOgogICAgLSBuYW1lOiAiZHVtcCIKICAgICAgZmlsZToKICAgICAgICBwYXRoOiAiL2Rldi9zdGRvdXQiCiAgICAgICAgbGF5b3V0OiB7fQogIHJvdXRlOgogICAgcm91dGVzOgogICAgICAtIG1hdGNoOgogICAgICAgICAgLSByZWNlaXZlcjogImR1bXAiCnJiYWM6CiAgY3JlYXRlOiB0cnVlCnNlcnZpY2VBY2NvdW50OgogIGNyZWF0ZTogdHJ1ZQogIGF1dG9tb3VudFNlcnZpY2VBY2NvdW50VG9rZW46IHRydWUKcG9kU2VjdXJpdHlDb250ZXh0OgogIGVuYWJsZWQ6IHRydWUKICBmc0dyb3VwOiAxMDAyCmNvbnRhaW5lclNlY3VyaXR5Q29udGV4dDoKICBlbmFibGVkOiB0cnVlCiAgcmVhZE9ubHlSb290RmlsZXN5c3RlbTogZmFsc2UKICBjYXBhYmlsaXRpZXM6CiAgICBhZGQ6IFtdCiAgICBkcm9wOgogICAgICAtIEFMTAogIHJ1bkFzVXNlcjogMTAwMgpleHRyYURlcGxveToKLSB8CiAgYXBpVmVyc2lvbjogYXBwcy92MQogIGtpbmQ6IERlcGxveW1lbnQKICBtZXRhZGF0YToKICAgIGxhYmVsczoKICAgICAgYXBwOiBrdWFyZAogICAgbmFtZToga3VhcmQtdmliCiAgICBuYW1lc3BhY2U6IHt7IC5SZWxlYXNlLk5hbWVzcGFjZSB9fQogIHNwZWM6CiAgICByZXBsaWNhczogMQogICAgc2VsZWN0b3I6CiAgICAgIG1hdGNoTGFiZWxzOgogICAgICAgIGFwcDoga3VhcmQKICAgIHRlbXBsYXRlOgogICAgICBtZXRhZGF0YToKICAgICAgICBsYWJlbHM6CiAgICAgICAgICBhcHA6IGt1YXJkCiAgICAgIHNwZWM6CiAgICAgICAgY29udGFpbmVyczoKICAgICAgICAtIGltYWdlOiBnY3IuaW8va3Vhci1kZW1vL2t1YXJkLWFtZDY0OjEKICAgICAgICAgIG5hbWU6IGt1YXJkLXZpYg==",
```

Comment:
You can reduce the number of `runtime_params` as you are not using the `extraDeploy` in the test (you are creating the Pod in Ginkgo already)

Reply:
Thanks! I forgot to update them before creating the PR 😄